dbbuild/sources/byrskabishop2022/util.py

```diff
@@ -131,14 +129,21 @@ def compile_frequencies(haplotypes):
 
 
 def list_frequencies(haplotypes):
+    admixed = ("ACB", "ASW", "CLM", "MXL", "PEL", "PUR")
     pop_tallies = defaultdict(lambda: defaultdict(Counter))
     agg_tallies = defaultdict(Counter)
     for n, row in haplotypes.iterrows():
         for haplokey in ("Haplotype1", "Haplotype2"):
             mhallele = [row[haplokey]]
             if not pd.isna(mhallele):
-                pop_tallies[row["Marker"]][row["Superpopulation"]].update(mhallele)
+                # The following line could arguably be moved into the conditional block below to
+                # excluded admixed individuals from the aggregate haplotype tallies. But as of
+                # today, I think including them in the aggregate totals is appropriate.
+                # -- DSS, 2023-02-28.
                 agg_tallies[row["Marker"]].update(mhallele)
+                pop_tallies[row["Marker"]][row["Population"]].update(mhallele)
+                if row["Population"] not in admixed:
+                    pop_tallies[row["Marker"]][row["Superpopulation"]].update(mhallele)
```

Comment:
...individuals from admixed population groups are retained and then filtered when MH alleles are tabulated.